pkg/restapi/acerp/operation/operations.go

```diff
@@ -186,6 +188,7 @@ func New(config *Config) (*Operation, error) {
 		homePageHTML:         config.HomePageHTML,
 		loginHTML:            config.LoginHTML,
 		dashboardHTML:        config.DashboardHTML,
+		portalHTML:           config.PortalHTML,
```

Comment:
Looks like this is not used. Can we remove this ?

Reply:
done